apps/root/src/state/transactions/transactionUpdater.tsx

```diff
@@ -234,6 +277,15 @@ export default function Updater(): null {
                 TransitionComponent: Zoom,
               }
             );
+
+            // TODO: What about transfering to own accounts?
```

Comment:
how you mean?

Reply:
Oops! I forgot about that.
I thought about a corner case involving transferring or swapping+transferring to one's own wallets. We would need to update the `to` token for the recipientAddress, if that address is loaded as one of the wallets